app/models/proceeding_type.rb

```diff
@@ -7,11 +7,15 @@ class ProceedingType < ApplicationRecord
 
   validates :code, presence: true
 
-  scope :with_ccms_code_starting, ->(_text) do
-    where(arel_table[:ccms_code].matches_regexp('^DA\d+$'))
-  end
-
   def self.populate
     ProceedingTypePopulator.call
   end
+
+  def default_substantive_scope_limitation
+    proceeding_type_scope_limitations.where(substantive_default: true).first.scope_limitation
+  end
+
+  def default_delegated_functions_scope_limitation
+    proceeding_type_scope_limitations.where(delegated_functions_default: true).first.scope_limitation
```

Comment:
would this work?
```suggestion
    proceeding_type_scope_limitations.find_by(delegated_functions_default: true).scope_limitation
```

Reply:
I've taken Rob's suggestion and delegated this to the join table